kube/services/jobs/usersync-cronjob.yaml

```diff
@@ -130,12 +135,21 @@ spec:
                   fi
                 else
                   if [[ -f /mnt/shared/user.yaml ]]; then
-                    fence-create sync --sync_from_dbgap $(SYNC_FROM_DBGAP) --projects /var/www/fence/projects.yaml --yaml /mnt/shared/user.yaml
+                    output=$(fence-create sync --sync_from_dbgap "$SYNC_FROM_DBGAP" --projects /var/www/fence/projects.yaml --yaml /mnt/shared/user.yaml)
                   else
-                    fence-create sync --sync_from_dbgap $(SYNC_FROM_DBGAP) --projects /var/www/fence/projects.yaml
+                    output=$(fence-create sync --sync_from_dbgap "$SYNC_FROM_DBGAP" --projects /var/www/fence/projects.yaml)
+                  fi
+                  exitcode=$?
+                  echo "$output"
+                  files=$(echo "$output" | grep "Reading file")
```

Comment:
can you comment about this is for us to know what files we pull from dbgap in slack